DAQ/src/ArtBinaryPacketsFromDigis_module.cc

```diff
@@ -0,0 +1,1113 @@
+//
+// EDProducer module for converting straw hit digis into DTC formatted
+// packets (which are stored in DataBlock data products)
+//
+//
+// Original author Tomonari Miyashita
+//
```

Comment:
Is Tomo really the original author of this?  It's appropriate to give him credit if this is substantially based on his work but if it's a total rewrite it's your code.